test/unit/index-test.js

```diff
@@ -394,18 +396,30 @@ test('emits ring changed event', function t(assert) {
 
     assertChanged(function assertIt() {
         ringpop.membership.makeFaulty(node1Addr);
+    }, {
+        adding: [],
+        removing: [node1Addr]
     });
 
     assertChanged(function assertIt() {
         ringpop.membership.makeAlive(node1Addr, magicIncNo);
+    }, {
+        adding: [],
+        removing: [node1Addr]
```

Comment:
This formulation makes the tests pass, but doesn’t match my intuition of what should happen in this case.